Snakefile

```diff
@@ -186,6 +186,7 @@ rule process_airport_data:
         passengers_data="data/airport_data/T100_Domestic_Market_and_Segment_Data_-3591723781169319541.csv",
     output:
         statewise_output="plots/results/passengers_vs_consumption.csv",
+        merged_data="plots/results/merged_data.csv",
```

Comment:
I would suggest maybe changing the name to `merged_data="plots/results/merged_airports.csv",`. What do you think? So it is clear what data it is and it can be used instead of `airports.csv` by removing `merged_` part.